clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobFinder.java

```diff
@@ -0,0 +1,9 @@
+package com.linkedin.davinci.blob;
```

Comment:
nit: do you think we can move these files to `venice-common` package?

Reply:
Specifically `BlobFinder`, `BlobPeersDiscoveryResponse`, as I plan to use them in `venice-common`

Reply:
Hmm, but I think they are all for Da Vinci and Server functionality right? Or did you place everything blob related in venice-common already?

Reply:
> Or did you place everything blob related in venice-common already?
Yes, I thought this module might be used by vpj so i think it was more appropriate to have them in `venice-common`. That said, I can move them to `da-vinci-client` package tho.

Reply:
OK, for simplicity, let us move all of them in the venice-common for now, so don't need to write duplicate unit test to pass coverage

Reply:
I moved them to venice-common